storybook/storybook-react/config/preview-head.html

```diff
@@ -1,3 +1,7 @@
 <script>
   window.global = window;
+
+  // Set language to Dutch for Canvas and Stories
+  document.documentElement.setAttribute("lang", "nl");
+  document.querySelectorAll(".docs-story").forEach((e) => e.setAttribute("lang", "nl"));
```

Comment:
This should work, right? Any idea why it doesn’t?

Reply:
I think stories are loaded after the document is loaded, so elements with the `docs-story` class don't exist yet on document load. I've added this script to the story `decorators`, that seems to work

Reply:
Does this also work when you open the frame directly?

Reply:
> although it fires too often – it queries the entire document for each Story on it

Derp, missed that. I've gone with another approach, simply wrapping the story in a div with the correct language. Should be more robust.

> Does this also work when you open the frame directly?

Yeah, the new approach does. The decorators also cascade properly, so the stories are wrapped in `<div lang="nl">` first, and any other decorators defined on the story level after.